client/mm/libxc/bitget.go

```
@@ -0,0 +1,3133 @@
+		return bg.marketSnapshot.m, nil
+	}
+
+	matches, err := bg.MatchedMarkets(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("error getting market list: %w", err)
+	}
+
+	// Group MarketMatches by Slug (like Binance)
+	// Use uppercase for case-insensitive matching with ticker symbols
+	mkts := make(map[string][]*MarketMatch, len(matches))
+	for _, m := range matches {
+		slugUpper := strings.ToUpper(m.Slug)
+		mkts[slugUpper] = append(mkts[slugUpper], m)
+	}
+
+	// Get 24hr ticker data
+	var resp bgtypes.APIResponse
+	err = bg.getAPI(ctx, "/api/v2/spot/market/tickers", nil, false, &resp)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.Code != "00000" {
+		return nil, fmt.Errorf("bitget API error: %s - %s", resp.Code, resp.Msg)
+	}
+
+	var tickers []*bgtypes.TickerData
+	if err := json.Unmarshal(resp.Data, &tickers); err != nil {
+		return nil, fmt.Errorf("error unmarshaling tickers: %w", err)
+	}
+
+	// Iterate through ticker responses, not MarketMatches (like Binance)
+	m := make(map[string]*Market, len(tickers))
+	for _, ticker := range tickers {
+		// Normalize to uppercase for case-insensitive matching
+		tickerSymbolUpper := strings.ToUpper(ticker.Symbol)
+		ms, found := mkts[tickerSymbolUpper]
+		if !found {
+			// Ticker returned but doesn't match any MarketMatch Slug - skip it
+			continue
+		}
+		// Create Market entry for EACH MarketMatch in this Slug group
+		for _, mkt := range ms {
+			baseMinWithdraw, quoteMinWithdraw := bg.minimumWithdraws(mkt.BaseID, mkt.QuoteID)
+
+			high24h, _ := strconv.ParseFloat(ticker.High24h, 64)
+			low24h, _ := strconv.ParseFloat(ticker.Low24h, 64)
+			close, _ := strconv.ParseFloat(ticker.Close, 64)
+			quoteVol, _ := strconv.ParseFloat(ticker.QuoteVol, 64)
+			baseVol, _ := strconv.ParseFloat(ticker.BaseVol, 64)
+			openUtc, _ := strconv.ParseFloat(ticker.OpenUtc, 64)
+			changeUtc24h, _ := strconv.ParseFloat(ticker.ChangeUtc24h, 64)
```

Rather not ignore errors.